src/plugins/gui/RewardsCardPlugin.tsx

```diff
@@ -42,13 +42,13 @@ export const makeRewardsCardPlugin: FiatPluginFactory = async params => {
 
   async function getRewardCards(): Promise<RewardsCardItem[]> {
     const giftCards = await provider.otherMethods.getGiftCards()
-    const rewardCards: RewardsCardItem[] = giftCards.map(card => {
+    const rewardCards: RewardsCardItem[] = giftCards.cards.map(card => {
       // Expires 6 calendar months from the creation date
       const expirationDate = new Date(card.CreatedDate.valueOf())
       expirationDate.setMonth(card.CreatedDate.getMonth() + 6)
 
       return {
-        id: card.Id.toString(),
+        id: card.Id,
         // Expires in 180 days (6 months) from creation date
         expiration: expirationDate,
         url: card.CardNumber
```

Comment:
This is actually a comment for line 58 I think the old cards that expire first should be at the top of the list.